deltachat-jsonrpc/src/api/types/webxdc.rs

```diff
@@ -72,7 +75,11 @@ impl WebxdcMessageInfo {
             source_code_url: maybe_empty_string_to_option(source_code_url),
             internet_access,
             self_addr,
-            send_update_interval,
+            send_update_interval: send_update_interval
+                .checked_add(Duration::from_micros(999))
+                .context("Overflow occurred")?
+                .as_millis()
```

Comment:
why is checked_add() needed here? 

to enforce correct rounding from milliseconds from microseconds?  this would deserve a comment.

again, i think, in this case just using a plain integer in rust as well as in js land is way easier, and also easier to understand. 

i agree that Duration is good if it is about calculations, but that is not the case here

Reply:
I moved this rounding code to `src/webxdc.rs` now. `Duration` is indeed probably not needed in the webxdc code, but it's needed in `Ratelimit` because it's a low-level Rust tool. With `Duration` we can even publish it to crates.io if we want, milliseconds aren't good for a crate interface